app/helpers/submissions_helper.rb

```diff
@@ -86,7 +86,9 @@ def get_submissions_table_info(assignment, groupings)
           g[:grace_credits_used] = grouping.grace_period_deduction_single
           g[:section] = grouping.section
         end
-        g[:name_url] = get_grouping_name_url(grouping, result)
+        g[:name_url] = assignment.is_peer_review? && current_user.is_a_reviewer?(assignment) ?
+            edit_assignment_result_path(assignment.parent_assignment.id, result_pr.result_id) :
+            get_grouping_name_url(grouping, result)
```

Comment:
Note: Needed to use the parent ID for this to work properly with the view. Don't know if this should be changed to using the PR assignment ID and updating everything else? Unsure how messy that might be.

If we drop use parent_assignment to just using assignment.id, a 404 error occurs when using the peer review id instead of the parent assignment's ID.